client/src/utils/chooserModal.js

```diff
@@ -0,0 +1,143 @@
+import $ from 'jquery';
```

Comment:
I know there is inconsistency for file names but I think more in this folder are `kebab-case.js` and not `camelCase.js` - can we move towards kebab-case for new things?

This is not documented anywhere but I have it listed on our UI team development docs questions to answer.

Reply:
Also, how come this has been added to `utils`? this may be more suitable in the new `includes` folder as it is not really a simple util.

Reply:
As far as I can see, kebab-case is only used in the `endpoints` subfolder, and most files there have been inherited from previous locations in `static_src` within app folders, with their original names intact, so I don't think we can reliably draw any conclusions about current naming conventions from there. On the other hand, "use camelCase for modules under `client` that are imported into other client-side code" seems to be the far more consistently followed convention.

Happy to move this to `includes` - would be good to have some formal documentation of what these folders mean, but comparing the other things in there it does indeed look like a better fit.

Reply:
To be honest - I have no idea either. I have added this to the potential frontend development docs as a question. 

@thibaudcolas maybe you can shed some light (or we can discuss at the UI meeting) - where stuff is meant to go, whether we want to work towards a clear folder structure and file naming.

Reply:
Now moved to client/src/includes in latest rebase.